readme.md

```diff
@@ -1,6 +1,13 @@
-1. We strive to create a welcoming and inclusive community for everyone, regardless of their race, ethnicity, gender, sexual orientation, religion, or any other personal identity.
-2. Bigotry, hate speech, harassment, or any other form fo discrimination will not be tolerated and will result in an immediate ban.
-3. Keep conversations in the appropriate channels. 
-4. No spamming
-5. Do not share personal information
-6. If you witness any rule violations or feel unsafe, please let a moderator know.
+Before joining our community, please take a moment to review and accept the following guidelines. We designed these rules to ensure Duduke provides a safe, respectful, and welcoming environment for all members, regardless of their race, background, or other personal characteristics.
+
+1. **Respect Others:** Always treat others with respect and kindness. Everyone has a right to feel safe and comfortable here.
+2. **No Discrimination or Harassment:** Any form of discrimination, harassment, or bullying based on race, ethnicity, nationality, religion, gender, sexual orientation, age, or any other personal characteristic, is strictly prohibited.
+3. **No Hate Speech or Offensive Language:** Hate speech, slurs, or any offensive language, images, or links are not allowed.; this includes any content that is racist, sexist, homophobic, transphobic, or otherwise offensive or discriminatory.
+4. **No NSFW Content:** This server is SFW (Safe for Work). Do not share or discuss explicit, violent, or otherwise NSFW content.
+5. **No Spamming or Flooding:** Do not flood the chat with messages, emojis, or pings; this disrupts conversation and may result in a temporary mute.
+6. **No Personal Information:** Do not share or ask for personal information, such as addresses, phone numbers, or social security numbers - this includes your information and others'.
+7. **No Advertising or Self-Promotion:** Do not promote your own or others' social media, businesses, or other projects without permission from the admins.
+8. **No Trolling or Griefing:** Do not intentionally provoke or annoy others; this includes trolling, griefing, or posting misleading links.
+9. **Report Rule-breaking:** If someone breaks these rules, please report them to a @Moderator. Do not engage with or escalate the situation yourself.
+
+Remember, we intend this server to be a safe and welcoming space for everyone. We may mute, kick, or ban you if you do not follow these rules. We appreciate your cooperation and look forward to building a great community together.
```

Comment:
Only a list of rules can be set up in Discord's "Rule Screening" page. So everything should be a list item.